src/applications/static-pages/health-care-manage-benefits/components/CernerCallToAction/index.js

```diff
@@ -246,8 +250,6 @@ export class CernerCallToAction extends Component {
                   className="vads-c-action-link--blue"
                   href={myHealtheVetLink}
                   onClick={onCTALinkClick}
-                  rel="noreferrer noopener"
```

Comment:
Curious - why were these removed?

Reply:
It's in the third acceptance criteria to open the page in the same tab

Reply:
Ah thank you! I missed that